temporal/scripts/generate_metadata.py

```diff
@@ -0,0 +1,94 @@
+# (C) Datadog, Inc. 2023-present
```

Comment:
Do we want to keep this file here or could we include it in ddev?

Reply:
I wouldn't move it to ddev since it's specific to this integration; and I would keep it because it might be useful for updating the metadata.csv file with less opportunities for mistakes.